docs/static/releasenotes.asciidoc

```diff
@@ -17,6 +18,45 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6-2-3]]
+=== Logstash 6.2.3 Release Notes
+
+* There are no changes to Logstash core in this release
+
+==== Plugins
+
+*Fluent codec*
+
+* Encode tags as fluent forward protocol tags
```

Comment:
Encoded

Reply:
Hmm... I pondered this before I wrote this entry. I wondered if 'Encoded' in the past tense sounds like a one off action rather than a change in behavior. 

Is there some prefix, such as 'Added ability to ...' that might work?

Reply:
Yes.  Much better wording with "Added ability to "